js/file/shared/FilePreview.svelte

```diff
@@ -33,9 +33,9 @@
 					</td>
 
 					<td class="download">
-						{#if file.data}
+						{#if file.data || file.blob}
```

Comment:
Why do we have blobs now? All files are uploaded/stored on the server. file.data should not be null 🤔